pym/bob/input.py

```diff
@@ -2463,6 +2489,9 @@ def __parse(self):
         self.__createSchemas()
 
         # user config(s)
+        self.__parseUserConfig("/etc/bobdefault.yaml")
```

Comment:
you can use `XDG_CONFIG_HOME/bob/default.yaml` or if XDG_CONFIG_HOME is not set `~/.config/bob/default.yaml` and add your alias.